packages/gatsby-plugin-now/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "gatsby-plugin-now",
+  "version": "1.2.0",
+  "main": "index.js",
+  "license": "MIT",
+  "scripts": {
+    "build": "./build.sh",
+    "test-integration-once": "jest"
+  },
+  "files": [
+    "index.js",
+    "gatsby-node.js"
+  ],
+  "peerDependencies": {
+    "gatsby": ">=2.0.0"
+  },
+  "devDependencies": {
+    "jest": "^24.9.0"
+  },
+  "jest": {
```

Comment:
Is this jest config necessary?

Reply:
Yes, it is necessary to avoid `jest` running tests that are auto generated by gatsby in the fixtures folder